modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -386,4 +454,22 @@ class Instrument_Manager extends \NDB_Menu_Filter
         }
         return 'Appears Valid';
     }
+
+    /**
+     * Return whether the quatUser is properly configured.
+     * The quatUser is a MySQL user with CREATE table permissions.
+     * `empty` is used instead of `isset` as blank values in the config file
+     * are still considered set.
+     *
+     * @return bool True if a value for quatUser and quatPassword exist
+     */
+    function isQuatUserConfigured() : bool
+    {
+        $db        = \Database::singleton();
+        $config    = \NDB_Config::singleton();
+        $db_config = $config->getSetting('database');
+
+        return !empty($db_config['quatUser'])
+            && !empty($db_config['quatPassword']);
+    }
```

Comment:
Should we ensure that the user/password combination is valid by actually opening a connection to the DB?